homeassistant/components/zwave_js/light.py

```diff
@@ -123,7 +130,13 @@ def __init__(
 
         # Entity class attributes
         self._attr_supported_features = 0
-        if self._dimming_duration is not None:
+        if (
```

Comment:
Just a thought, you use these same conditions (checking if brightness transitions are supported and checking if color transitions are supported), might be cleaner to make class variables that contain that logic and then reference them twice.

e.g. 
```
self.supports_brightness_transition = bool(
    self._target_value is not None
    and "transitionDuration" in self._target_value.metadata.value_change_options
)
self.supports_color_transition = bool(
    self._target_color is not None
    and "transitionDuration" in self._target_value.metadata.value_change_options
)

then `if self.supports_brightness_transition or self.supports_color_transition` etc.
```

Separately, can we rename `self._target_value` to `self._target_brightness`? That will make things more readable